src/construction.cpp

```diff
@@ -1520,8 +1520,12 @@ void finalize_constructions()
         }
     }
 
+    // previously dead code became live in #17451 and caused #17443 and #17449
+    // is temporarily disabled pending refactor as construction_id = string_id<construction>
+    /*
```

Comment:
The git history doesn't provide much context however. That was previously a global function that I inlined as part of this PR.